### ISSUE TITLE
Correct namespace in magnetic field maps 

### ISSUE THREAD

**Issue by bovulpes** [AliceO2Group/AliceO2#334]
Hello,

The magnetic field maps in Common/maps/mfchebKGI_sym.root must be recreated (I could do it but I don't know how) after the change of the namespace AliceO2 ---> o2

**Comment by sawenzel** [comment]
@shahor02: Could you take a look, please? ... or let me know how this is done.

**Comment by shahor02** [comment]
Hi Bogdan,

Yes, it was screwed up by the renaming of the class, I'll fix this today (need to go via intermediate dummy class).

Cheers,
 Ruben

**Comment by sawenzel** [comment]
This shows again that we must get the unit tests running during PR checks ... (see #294). Once this is done these kind of errors can be avoided.

**Comment by sawenzel** [comment]
Hi, ROOT should be able to solve this issue. We need to add a schema evolution rule to our dictionary generation ... that maps the source class written in the file (aka AliceO2::Field) to the new in-memory target class (aka o2::Field).

**Comment by shahor02** [comment]
Hi @sawenzel , thanks, but I think this method is not convenient in out case: it allows to map different versions of the same class or event different classes to each other. But these classes should be known to the system at compilation time. Therefore we will need to maintain both AliceO2::Field and o2::field versions (in fact for 3 classes, since 2 helper classes for Cheb.param. were also renamed)
Instead, I've revived the old method to write and read from text file, so will regenerate it for new namespace. 

This will cost upload of 9MB binary file to git, but I consider this as temporary solution: eventually this file should go wherever all binary files will go.

**Comment by sawenzel** [comment]
@shahor02 : As far as I understand the old class does not need to be known at compile time. It is the class written to the root file (which contains the streamerinfo and hence has all the knowledge).

**Comment by shahor02** [comment]
@sawenzel : Perhaps this will work, but trying testing this means essentially writing 3 copy constructors (with dynamic memory allocation etc) to the linkdef.h, while the conversion via intermediate file is already done...

**Comment by sawenzel** [comment]
@bovulpes : Can you check if it works now and eventually close the issue?

**Comment by bovulpes** [comment]
It works, thanks, MFT simulation/reconstruction is back.

**Comment by sawenzel** [comment]
P.S. I tried out the solution with schema evolution, and this also works if we use the type-safe "File::GetObject" method instead of "File::Get".

**Comment by shahor02** [comment]
Thanks @sawenzel , do you mean you implement a "copy code" for o2::field... from Alice02:Field... for the full hierarchy of classes involved in the field calculation, or that one could use {{#pragma}} rules even for classes not known at compilation time?

**Comment by sawenzel** [comment]
The later: Just adding a few pragmas and using the 'File::GetObject' method ...
